packages/widget/src/pages/SelectWalletPage/SelectWalletPage.tsx

```diff
@@ -61,7 +67,24 @@ export const SelectWalletPage = () => {
           paddingRight: 1.5,
         }}
       >
-        {supportedWallets.map((wallet: Wallet) => (
+        {installedWallets.map((wallet: Wallet) => (
+          <ListItemButton
+            key={wallet.name}
+            onClick={() => handleConnect(wallet)}
+          >
+            <ListItemAvatar>
+              <Avatar
+                src={(wallet.icon as any).src || wallet.icon}
+                alt={wallet.name}
+              >
+                {wallet.name[0]}
+              </Avatar>
+            </ListItemAvatar>
+            <ListItemText primary={wallet.name} />
+          </ListItemButton>
+        ))}
+
+        {notInstalledWallets.map((wallet: Wallet) => (
```

Comment:
We can probably merge these arrays and map through one. 🙂

Reply:
true

Reply:
fixed